corehq/apps/app_manager/templates/app_manager/form_designer.html

```diff
@@ -28,15 +28,7 @@
       padding-bottom: 0;
       margin-bottom: 0;
     }
-    .custom-button-style {
-      border: 1px solid #ccc;
-      padding: 5px 10px;
-      margin-right: 10px;
-    }
-    .custom-button-style:hover {
-      border-color: #999;
-    }
-    .btn-group > .custom-button-style {
+    .btn-group > .btn {
```

Comment:
this should be `.btn-group > .btn-form-action`, otherwise this is overriding the styles from bootstrap which is a no-no. Did you take a look at whether `btn-group` needed to be there? the whole point of button group is to remove the border radius!